glyphs-reader/src/font.rs

```diff
@@ -953,6 +978,59 @@ impl TryFrom<RawGlyph> for Glyph {
     }
 }
 
+// https://github.com/googlefonts/glyphsLib/blob/24b4d340e4c82948ba121dcfe563c1450a8e69c9/Lib/glyphsLib/builder/features.py#L43
+fn autostr(feature: &RawFeature) -> String {
+    match feature.automatic {
+        Some(1) => "# automatic\n".to_string(),
+        _ => "".to_string(),
+    }
+}
+
+fn name(feature: &RawFeature) -> Result<String, Error> {
```

Comment:
"name" is quite a short and common _name_ to name a function, may inadvertently be confused with some other local variable named like that, maybe rename "feature_name"?

Reply:
I feel like name(RawFeature) is arguably clear though I buy the consumption of a common name. Let me just add it directly to RawFeature so it's feature.name()

Reply:
Fixed